valhalla/sif/dynamiccost.h

```diff
@@ -212,6 +202,19 @@ class DynamicCost {
            (ignore_oneways_ && (edge->reverseaccess() & access_mask_));
   }
 
+  /*
+   * Determine whether an edge is currently closed due to traffic.
+   * @param  edgeid         GraphId of the opposing edge.
+   * @return  Returns true if the edge is closed due to live traffic constraints, false if not.
+   */
+  inline virtual bool IsClosedDueToTraffic(const baldr::GraphId& edgeid,
```

Comment:
this should be a private/protected function, there is no need to have this in the public interface

Reply:
yeah, make sense